test-case-reporting/tsconfig.json

```diff
@@ -21,6 +21,7 @@
     "./*",
     "src/**/*",
     "test/*",
-    "types/*"
+    "types/*",
+    "test/fixtures/*"
```

Comment:
You can include all files under `test/` with `"test/**/*"`

Reply:
Done.